fcli-other/fcli-functional-test/src/ftest/groovy/com/fortify/cli/ftest/ssc/SSCJobSpec.groovy

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2023 Open Text.
+ *
+ * The only warranties for products and services of Open Text 
+ * and its affiliates and licensors ("Open Text") are as may 
+ * be set forth in the express warranty statements accompanying 
+ * such products and services. Nothing herein should be construed 
+ * as constituting an additional warranty. Open Text shall not be 
+ * liable for technical or editorial errors or omissions contained 
+ * herein. The information contained herein is subject to change 
+ * without notice.
+ */
+package com.fortify.cli.ftest.ssc
+
+import static com.fortify.cli.ftest._common.spec.FcliSessionType.SSC
+
+import com.fortify.cli.ftest._common.Fcli
+import com.fortify.cli.ftest._common.spec.FcliBaseSpec
+import com.fortify.cli.ftest._common.spec.FcliSession
+import com.fortify.cli.ftest._common.spec.Prefix
+import com.fortify.cli.ftest.ssc._common.SSCAppVersion
+
+import spock.lang.AutoCleanup
+import spock.lang.Requires
+import spock.lang.Shared
+
+@Prefix("ssc.job") @FcliSession(SSC) 
+class SSCJobSpec extends FcliBaseSpec {
+    
+    def "list"() {
+        def args = "ssc job list --store jobs"
+        when:
+            def result = Fcli.run(args)
+        then:
+            verifyAll(result.stdout) {
+                size()>0
+                it[0].replace(' ', '').equals("JobnameJobgroupJobclassStateCancellablePriorityCreatetimeStarttimeFinishtime")
+                it.any { it.startsWith(" JOB") }
+            }
+    }
+    
+    def "extractJobName"() {
```

Comment:
I guess this is meant to extract a job name that can then be used in the next test (as the TODO on the get.byName test will need to be fixed in order for that test to succeed on other SSC instances)? Some feedback:
- This test assumes that the list test is executed first, so this spec needs to have the `@StepWise` annotation
- I don't remember whether you can use SpEL expressions in variable references, but potentially you could simply use `::jobs::get(0),jobName` in the get command, instead of having this intermediate test
- If the above doesn't work, potentially you can do something with the `-o expr=...` option to output only the job name, so you can simply use the contents of `result.out` as input for the `get` command